apps/mobile/app/(app)/(tabs)/index.tsx

```diff
@@ -7,21 +7,33 @@ import {
   ActivityIndicator,
 } from "react-native";
 import { useAuth } from "../../../lib/AuthContext";
-import { useEffect, useState } from "react";
-import { actionsFindAll, ActionDto } from "../../../../../shared/client";
+import { useEffect, useMemo, useState } from "react";
+import { ActionDto } from "../../../../../shared/client";
+import { actionsFindAllWithStatus } from "../../../../../shared/client";
 import ActionCard from "../../../components/ActionCard";
 import { router } from "expo-router";
+import { FilterMode, filterActions } from "../../../../../shared/lib/actionUtils";
+
+const FILTERS = [
+  FilterMode.All,
+  FilterMode.Active,
+  FilterMode.Upcoming,
+  FilterMode.Past,
+  FilterMode.Joined,
+];
```

Comment:
This still feels like a redefinition that should be in /shared? If you need it as an array I think you can do `Object.keys`?